### PR TITLE
HCIDOCS-129: Fix typo in IPI/bare metal install

### PR DESCRIPTION
**Pull request by kdrake20**
**Fixes:** [HCIDOCS-129](https://issues.redhat.com/browse/HCIDOCS-129)

**For:** Version 4.12+

[**Preview**](https://76471--ocpdocs-pr.netlify.app)

**Signed-off-by:** Katie Drake kdrake@redhat.com

### CODE REVIEW

File: modules/ipi-install-configuring-the-install-config-file.adoc [hunk 1]
```diff
@@ -117,7 +117,7 @@ ironic-inspector inspection failed: No disks satisfied root device hints
 
 [NOTE]
 ====
-Before {product-title} 4.12, the cluster installation program only accepted an IPv4 address or and IPv6 address for the `apiVIP` and `ingressVIP` configuration settings. In {product-title} 4.12 and later, these configuration settings are deprecated. Instead, use a list format in the `apiVIPs` and `ingressVIPs` configuration settings to specify IPv4 addresses, IPv6 addresses or both IP address formats.
+Before {product-title} 4.12, the cluster installation program only accepted an IPv4 address or an IPv6 address for the `apiVIP` and `ingressVIP` configuration settings. In {product-title} 4.12 and later, these configuration settings are deprecated. Instead, use a list format in the `apiVIPs` and `ingressVIPs` configuration settings to specify IPv4 addresses, IPv6 addresses or both IP address formats.
```
Comment (aspauldi) [inline modules/ipi-install-configuring-the-install-config-file.adoc:120]: Suggestion to add a comma after 'IPv6 addresses' to comply with comma use for items in a series style mentioned in the [IBM Style Guide](https://www.ibm.com/docs/en/ibm-style?topic=punctuation-commas#commas-between-items-in-a-series). (I know you didn't change this in the PR, but I thought I'd mention it anyway.)

**Review by aspauldi — APPROVED** [review]
One tiny thing, @kdrake20. Thanks!

File: modules/ipi-install-configuring-the-install-config-file.adoc (hunk 1 above)
Comment (kdrake20) [reply modules/ipi-install-configuring-the-install-config-file.adoc]: Updated As Suggested.

**Comment by apinnick** [comment]
@mburke5678 This might be a dumb question, but why are we maintaining a 4.12 deprecation note in 4.16? Does it make sense to remove it after 4.14? I just noticed that this note appears in 5 different modules.

**Pull request merged**